src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
```

Comment:
Consider using `const` instead of `let` for `finalPrice` since its value is not reassigned after initialization.